teamContracts.md

```
@@ -0,0 +1,69 @@
+
+**Team Member Signatures:**  
+Raymond
+
+Kahn  
+
+Ethan
+
+Alec
+
+Hiu Yan
+_______________________________________  
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: ---; Actual: _______________________________________  

(MD035, hr-style)